src/backend/distributed/deparser/deparse_function_stmts.c

```diff
@@ -278,6 +278,16 @@ AppendDefElemSet(StringInfo buf, DefElem *def)
 {
 	VariableSetStmt *setStmt = castNode(VariableSetStmt, def->arg);
 
+	AppendVariableSet(buf, setStmt);
+}
+
+
+/*
+ * AppendVariableSet appends a string representing the VariableSetStmt to a buffer
+ */
+void
+AppendVariableSet(StringInfo buf, VariableSetStmt *setStmt)
+{
```

Comment:
Asserting that `setStmt` is not `NULL` could be beneficial before doing something with it

Reply:
Instead of using assertions here, I used `castNode()` in callers to make sure the parameters are not null